### DISCUSSION TITLE
Hardcoded Runes

### DISCUSSION THREAD

**Discussion by dannydeezy** [ordinals/ord#3103] [General]
I'm not a big fan of the hardcoded runes concept.
It's cute, but it really doesn't solve any of the problems it claims to solve - it just punts the issue to the minters - now everyone will be trying to mint in the first block (rather than deploy). And **still** people are going to be rushing to be the first "public" rune deployed.

**Comment by cliobitcoinbank** [comment]
Punting to the minters raises the cost of cornering the market vs cornering the market on firstie runes with a single pair of etch and mint functions.
Disrupting the usual patterns of failure is the first step to allowing runes to reach a different endgame than any other memecoin casino ever has.  Ordinals is too idealistic and purity driven, its a fault of ordinals not a strength and any such orthodoxy just becomes a cudgel to promote incumbents anyway.  Watching every scheming frontrunner get frontrun by the protocol itself has been an absolute joy.  Would-be plunderers crying they were scammed by fairness measures.  I practiced during the jubilee block so I could better snipe runes in the halvening, it's sick, I want this race to the bottom to end.

**Comment by raphjaph** [comment]
Basically the incentive for big capital allocators is to create the first runes as closed/premined and then sell to rune traders on secondary markets. We want people to actually mint these runes and therefore the first ten will all be open mints/etchings. Also there are many parameters that you can set on runes and we want to showcase a lot of them so people don't make the first ones boring corporate coins.

**Reply by dannydeezy** [reply]
thanks for the explanation - that does make sense. my other instinctual reaction was that having 10 (instead of just 1, or 2, or 3) makes it so there's no real Schelling point, which may decrease the expected success of any of them. but I do see the value and fun involved in showcasing different rune types

**Reply by JussCubs** [reply]
I think if a team / project is following the github and testing on testnet the unique parameters you want to showcase, that you should work with them to allow them to be included in this first 10. At least allow that option to be voted on by the full community not just hard code 10 and not give any other option for consensus for the decision. As long as the facts about whether a candidate seems like a corporate boring coin are well presented, then the community will decide the right decision. Hard coding anything is already dissociating decentralization, of which Bitcoin and Satoshi's vision was built upon. I understand your reasonings, but I think there should be a middle ground.

**Comment by raphjaph** [comment]
I'm going to close this so this can be moved over to https://github.com/casey/runestone
